SoloProjects/clc80/Dashboard/Dashboard/DashboardCard.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  DashboardCard.swift
+//  Dashboard
+//
+//  Created by Claudia Maciel on 12/13/20.
+//
+
+import UIKit
+
+class DashboardCard: UIView {
+    //MARK: - Properties
+    var image1 = UIImageView()
+    var image2 = UIImageView()
+    let title = UILabel()
+    let subTitle = UILabel()
+    
+    override init(frame: CGRect) {
+        super .init(frame: frame)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    init(image1: UIImageView, image2: UIImageView, title: String, subTitle: String) {
+        super .init(frame: .zero)
+        self.image1 = image1
+        self.image2 = image2
+        self.title.text = title
+        self .subTitle.text = subTitle
+        configure()
+    }
+    
+    private func configure() {
+        layer.cornerRadius = 10
+        self.backgroundColor = .systemPink
+        
+        self.addSubview(image1)
+        self.addSubview(image2)
+        self.addSubview(title)
+        self.addSubview(subTitle)
+        
+        translatesAutoresizingMaskIntoConstraints = false
+        image1.translatesAutoresizingMaskIntoConstraints = false
+        image2.translatesAutoresizingMaskIntoConstraints = false
+        title.translatesAutoresizingMaskIntoConstraints = false
+        subTitle.translatesAutoresizingMaskIntoConstraints = false
+     
+        title.font = UIFont.preferredFont(forTextStyle: .title1)
+        subTitle.font = UIFont.preferredFont(forTextStyle: .subheadline)
+        subTitle.textColor = .gray
+        
+        NSLayoutConstraint.activate([
```

Comment:
Nice, using `NSLayoutConstraint.activate` and the anchor APIs. I like to use these, too, I think they're a good blend of ease-of-use and familiarity.